sr_ethercat_hand_config/package.xml

```diff
@@ -20,6 +20,7 @@
   <!-- Dependencies needed to compile this package. -->
 
   <!-- Dependencies needed after this package is compiled. -->
+  <run_depend>sr_logging_common</run_depend>
```

Comment:
Actually, let's remove this dependency, as I see that we're not specifying runtime dependencies for this package, due to the fact that in sr-config we put launchfiles that are changing with each delivery.